src/index.html

```diff
@@ -20,7 +20,73 @@
         <![endif]-->
 
         <!-- Add your site or application content here -->
-        <p>Hello world! This is HTML5 Boilerplate.</p>
+        <header>
+        </header>
+
+        <section class="leftcol">
+          <img id="octocat" src="https://avatars3.githubusercontent.com/u/583231?v=3&s=400" />
+          <h1>The Octocat</h1>
+          <h2>octocat</h2>
+
+          <ul>
+            <li>GitHub</li>
+            <li>San Francisco</li>
+            <li>email link</li>
+            <li>website</li>
+            <li>Joined on Jan 25, 2011</li>
+          </ul>
+
+          <span class="numbers">
+            <h3>1.1k</h3>
+            <h3>2</h3>
+            <h3>6</h3>
+          </span>
+
+          <span class="category">
+            <h5>Followers</h5>
+            <h5>Starred</h5>
+            <h5>Following</h5>
+          </span>
```

Comment:
I think this text actually belongs with the `.numbers` above... If you check out the Reference Implementation, _both_ the numbers _and_ the words underneath are a part of the same link... Maybe you could put them together and make them `<big>`...?